src/Sulu/Bundle/ContentBundle/Resources/public/js/components/teaser-selection/list/main.js

```diff
@@ -96,6 +96,10 @@ define(['underscore'], function(_) {
             this.sandbox.on('husky.datagrid.teaser-selection.item.deselect', function(id) {
                 this.options.deselectCallback({type: this.options.type, id: id});
             }.bind(this));
+        },
+
+        getUrl: function() {
+            return this.options.url.replace('{locale}', this.options.url);
```

Comment:
the list is not used in the sulu kernel. only the article bundle uses it - but it can be used by a lot of custom stuff.